### DISCUSSION TITLE
Message attachments

### DISCUSSION THREAD

**Discussion by obrunsmann** [spatie/laravel-slack-alerts#5] [Ideas]
Awesome new package :)

Would love to have a way to add [message attachments](https://api.slack.com/reference/messaging/attachments) easily.

**Comment by Nielsvanpach** [comment]
Message attachments are a legacy part of the messaging functionality for Slack apps. Probably not going to support this with this package.

**Reply by obrunsmann** [reply]
Oh yes, you're right. Did not see that. So blocks are way more complicated and as you already stated, you won't support that in this package. Then I'll close this :)
